hack/verify-conversions.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+DESCHEDULER_ROOT=$(dirname "${BASH_SOURCE}")/..
+_tmpdir="$(mktemp -d "${DESCHEDULER_ROOT}/_tmp/kube-verify.XXXXXX")"
+
+_deschedulertmp="${_tmpdir}"
+mkdir -p "${_deschedulertmp}"
+
+git archive --format=tar --prefix=descheduler/ "$(git write-tree)" | (cd "${_deschedulertmp}" && tar xf -)
+_deschedulertmp="${_deschedulertmp}/descheduler"
+
+pushd "${_deschedulertmp}" > /dev/null 2>&1
+export PRJ_PREFIX="${_deschedulertmp}"
+echo "PRJ_PREFIX=: ","${PRJ_PREFIX}"
+hack/update-generated-conversions.sh
+popd > /dev/null 2>&1
+
+ret=0
```

Comment:
Where it the codes does this get set to something other than 0?

Reply:
I've removed `ret=0` with latest push, was using mainly to collect last status.